styleguide/components/31-molecules/teasers/teaser/_teaser.scss

```
@@ -0,0 +1,342 @@
+    font-weight: 700;
+  }
+}
+
+//
+// Contact teaser variant.
+//
+.teaser.teaser-contact {
+  @include theme('border-color', 'color-primary--lighten-4', 'teaser-contact-border');
+
+  padding-bottom: .7rem;
+  border-bottom: 2px solid;
+
+  &,
+  & + & {
+    margin-top: 1.2rem;
+  }
+
+  // @TODO: Move to other layer.
+  .ol-popup & {
+    padding-bottom: 0;
+    border-bottom: 0;
+  }
+
+  .content__bottom {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    > *:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &-info {
+    @extend %list-no-style;
+
+    margin-top: -$link-padding;
+    margin-bottom: .4rem;
+
+    li > .no-icon {
+      font-weight: normal;
+    }
+  }
+
+  .teaser-title-link {
+    margin-bottom: .8rem;
+    font-size: .8rem;
+    line-height: 1.2rem;
+  }
+
+  .read-more {
+    margin-top: auto;
+  }
+
+  // @TODO: Move to other layer.
+  .grid-3 & {
+    &,
+    .content__bottom {
+      height: 100%;
+    }
+  }
+}
+
+//
+//
+//
+.teaser.teaser-search,
+.teaser.teaser-wide,
+.teaser.teaser-search-wide {
+  @include theme('border-color', 'color-primary--lighten-4', 'view-heading-2-border-color');
+
+  padding-top: $gutter-width;
+  padding-bottom: $gutter-width;
+  border-bottom: 2px solid;
+
+  &:first-of-type {
+    @include theme('border-color', 'color-primary--lighten-4', 'view-heading-2-border-color');
+
+    margin-top: $gutter-width;
+    border-top: 2px solid;
+  }
+
+  dl dd {
+    margin: 0;
+    padding-left: .5rem;
+  }
+
+  @include tablet {
+    dt,
+    dd {
+      display: inline;
+    }
+  }
+
+  .tag,
+  .icon-list {
+    margin-bottom: .4rem;
+  }
+}
+
+.teaser.teaser-wide,
+.teaser.teaser-search-wide {
+  .teaser-content {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    @include tablet {
+      flex-wrap: nowrap;
+    }
+  }
+
+  .accolade-inverse.bottom-left {
+    display:  block;
+
+    @include tablet {
+      display: none;
+    }
+  }
+
+  .accolade-inverse.right-top {
+    display:  none;
+
+    @include tablet {
+      display: block;
+    }
+  }
+
+  .content__left,
+  .content__right {
+    flex-basis: 100%;
+  }
+
+  .content__left {
+    order: 1;
+    margin-bottom: gutter();
+
+    @include tablet {
+      flex-basis: span(2 of 9);
+      margin-right: gutter();
+      margin-bottom: 0;
+    }
+  }
+
+  .content__right {
+    order: 2;
+
+    @include tablet {
+      flex-basis: span(7 of 9);
+    }
+
+    .location {
+      font-weight: 700;
+
+      i {
+        margin-right: .3rem;
+        font-size: 1.5em;
+        vertical-align: middle;
+      }
+    }
+
+    .links {
+      margin-bottom: .4rem;
+
+      li {
+        margin-right: 0;
+
+        &::after {
+          margin-right: .4rem;
+          margin-left: .4rem;
+          content: '—';
+        }
+
+        &:last-of-type::after {
+          content: '';
+        }
+      }
+    }
+  }
+}
+
+//
+// Topic teaser variant.
+//
+.teaser.teaser-topic {
```

You could rename this to something more generic...

```suggestion
.teaser.teaser-underlined {
```

(it's currently only used in stad-gent so refactoring is easily done)

---

Not sure, that's ux / technical again.
Back in the old days, we only had two teasers: article and topic

And the layout referred to a specific function: no image -> overview page, image -> detail page.

Not sure how much of that logic remains in the new design.

---

I think underlined is okay... it's better that then topic...

---

That's like using style in your classnames: 'button.red'
One day, the style will change and your teaser-underlined will no longer be underlined.

---

There's not really a general rule in the usage of the teasers right now. Sometimes they have an image, sometimes they don't. There's also a variant with both an image and an underlining.

As far as the naming is concerned: 
- `topic` seems kind of random
- `underlined` might indeed be too specific, then again. `underlined` is a modifier class. All it does is add additional styling. If there ever is another variant of the teaser, you might as well add another modifier class... 

I vote `underlined`. 🎉